backend/tests/hmc-course.test.ts

```diff
@@ -2,10 +2,46 @@ import { describe, test, expect } from "@jest/globals";
 import type * as APIv4 from "hyperschedule-shared/api/v4";
 
 import { linkCourseData, extractSectionTerm } from "../src/hmc-api/course";
+import { parseCourseBoomi } from "../src/hmc-api/files";
 import * as fs from "fs/promises";
 
 const PARSED_SAMPLE_PATH = "src/hmc-api/sample/parsed-sample-v4.json";
 
+const BOOMI_DB_SAMPLE = `
```

Comment:
i wonder whether you should put this under a separate file instead of pasted inline. i don't care very strongly though.